Jenkinsfile_k8s

```diff
@@ -1,3 +1,5 @@
+@Library('pipeline-library@pull/252/head') _
```

Comment:
Specified to use a specific pipeline library version I presume?

Reply:
absolutely, and then remove on the main branch afterward.
It allowed to test https://github.com/jenkins-infra/pipeline-library/pull/252/ before merging it.

The exit code 137 we had on infra.ci was due to the pod being OOM killed